lib/active_public_resources/drivers/youtube.rb

```diff
@@ -120,7 +120,7 @@ def next_criteria(request_criteria, results)
       def parse_video(item)
         video_id = item['id']
         snippet = item['snippet']
-        statistics = item['statistics']
+        statistics = item['statistics'] ? item['statistics'] : {}
```

Comment:
I think this is the simplest way to handle the case, I'd rather not raise an exception if the key isn't found or supply a default (would require extra code)